mobile-widgets/qmlmanager.cpp

```diff
@@ -155,6 +155,8 @@ void QMLManager::openLocalThenRemote(QString url)
 		if (informational_prefs.unit_system == IMPERIAL)
 			informational_prefs.units = IMPERIAL_units;
 		prefs.units = informational_prefs.units;
+		prefs.tankbar = git_prefs.tankbar;
+		prefs.dcceiling = git_prefs.dcceiling;
```

Comment:
OK, so the good news is that my last comment on the previous commit was wrong.
I'm guessing you are doing it this way to avoid the timing challenge (i.e., are the prefs overwritten by something else, after we read from git? Or more likely, you are mirroring what someone else did with the "informational_prefs". But then it seems to me it would be best to just have "informational_prefs" turn into "git_prefs" and have one structure for both of these types of settings, don't you think?